geode-assembly/src/integrationTest/java/org/apache/geode/management/internal/rest/CommandForwardIntegrationTest.java

```diff
@@ -0,0 +1,25 @@
+package org.apache.geode.management.internal.rest;
```

Comment:
- add license header to CommandForwardIntegrationTest.java
- run ./gradlew spotlessApply